src/OpenRiaServices.Tools.TextTemplate/Test/TestHelper.cs

```diff
@@ -191,7 +195,7 @@ private static MethodInfo GetMatchingMethod(MethodInfo[] methods2, MethodInfo me
             ParameterInfo[] method1Params = method1.GetParameters();
             foreach (MethodInfo m in methods2.Where(m => m.Name == method1.Name))
             {
-                if (!AreTypesEqual(m.ReturnType, method1.ReturnType))
+                if (m.ReturnType.Name != method1.ReturnType.Name)
```

Comment:
1.Maybe "FullName" should be used to compare types
2. Should AreTypesEqual be fixed to compare by full name ? (depends on how much it is used in other places)